### PR TITLE
Fix calling instance method as static method

### PR DESCRIPTION
**Pull request by idetinkin**
## Type

- Non critical bugfix

### CODE REVIEW

File: src/Frontend/Core/Engine/Url.php [hunk 1]
```diff
@@ -257,7 +257,7 @@ private function determineLanguage(string $queryString): string
     private function setLanguageCookie(string $language): void
     {
         try {
-            self::getContainer()->get('fork.cookie')->set('frontend_language', $language);
+            $this->getContainer()->get('fork.cookie')->set('frontend_language', $language);
```
Comment (jessedobbelaere) [inline src/Frontend/Core/Engine/Url.php:260]: Actually, when we switch to Symfony 4 we need to get the container again from a static property... https://symfony.com/blog/new-in-symfony-4-1-simpler-service-testing

File: src/Backend/Modules/Analytics/Tests/Actions/IndexTest.php [hunk 2]
```diff
@@ -9,7 +9,7 @@ class IndexTest extends BackendWebTestCase
 {
     public function testAuthenticationIsNeeded(Client $client): void
     {
-        self::assertAuthenticationIsNeeded($client, '/private/en/analytics/index');
+        $this->assertAuthenticationIsNeeded($client, '/private/en/analytics/index');
```
Comment (jessedobbelaere) [inline src/Backend/Modules/Analytics/Tests/Actions/IndexTest.php:12]: 👍  I like instance methods for phpunit, like Symfony describes in the docs https://symfony.com/doc/current/testing.html#unit-tests

File: src/Frontend/Modules/Tags/Tests/Engine/ModelTest.php [hunk 3]
```diff
@@ -173,6 +173,6 @@ private function assertModuleTags($items): void
         self::assertArrayHasKey('name', $items);
         self::assertArrayHasKey('label', $items);
         self::assertArrayHasKey('items', $items);
```
Comment (jessedobbelaere) [inline src/Frontend/Modules/Tags/Tests/Engine/ModelTest.php:175]: But if we change all the static methods to instance methods for phpunit, we should change them all, not leave the tests in a half state of mixed statics with instance methods...

**Comment by carakas** [comment]
I'd rather move them all to static in Fork 6

**Comment by carakas** [comment]
Closing since we'll have a look at it in restructure code

**Pull request closed without merging**